cli/src/utils/downloadBin.ts

```diff
@@ -5,48 +5,6 @@ import * as os from "os";
 
 const fileExists = promisify(fs.exists);
 
-export const anchorBinUrlMap = new Map([
-  [
-    "linux-amd64",
-    "https://github.com/Lightprotocol/anchor/releases/download/v0.27.0/light-anchor-linux-amd64",
-  ],
-  [
-    "macos-amd64",
-    "https://github.com/Lightprotocol/anchor/releases/download/v0.27.0/light-anchor-macos-amd64",
-  ],
-  [
-    "macos-arm64",
-    "https://github.com/Lightprotocol/anchor/releases/download/v0.27.0/light-anchor-macos-arm64",
-  ],
-  [
-    "linux-arm64",
-    "https://github.com/Lightprotocol/anchor/releases/download/v0.27.0/light-anchor-linux-arm64",
-  ],
-]);
-
-export const macroCircomBinUrlMap = new Map([
-  [
-    "linux-amd64",
-    "https://github.com/Lightprotocol/macro-circom/releases/download/v0.1.1/macro-circom-linux-amd64",
-  ],
-  [
-    "macos-amd64",
-    "https://github.com/Lightprotocol/macro-circom/releases/download/v0.1.1/macro-circom-macos-amd64",
-  ],
-  [
-    "macos-arm64",
-    "https://github.com/Lightprotocol/macro-circom/releases/download/v0.1.1/macro-circom-linux-arm64",
-  ],
-  [
-    "linux-arm64",
-    "https://github.com/Lightprotocol/macro-circom/releases/download/v0.1.1/macro-circom-macos-arm64",
-  ],
-]);
-function replaceVersionTagInUrl(url: string, newTag: string): string {
-  // Matches the version pattern (e.g., v0.1.1) in the URL
-  const versionPattern = /v\d+\.\d+\.\d+/g;
-  return url.replace(versionPattern, newTag);
-}
 async function latestRelease(owner: string, repo: string) {
   const GITHUB = "https://api.github.com";
   console.log(`${GITHUB}/repos/${owner}/${repo}/releases/latest`);
```

Comment:
is the log wanted here?

Reply:
I changed the log message a bit, now it's:

```
console.log(`Checking the latest release of ${github}/repos/${owner}/${repo}/releases/latest`);
```

I think having it is useful for debugging, if anything goes wrong with checking releases.